app/models/work.rb

```diff
@@ -34,19 +34,20 @@ class Work < ApplicationRecord
     end
 
     event :begin_deposit do
-      transition first_draft: :depositing, version_draft: :depositing, pending_approval: :depositing
+      transition %i[first_draft version_draft pending_approval] => :depositing
```

Comment:
This is pure refactoring, using an array to indicate the before states all transition to the after state.